code/drasil-docLang/Drasil/ExtractDocDesc.hs

```diff
@@ -15,10 +15,9 @@ egetDocSec IntroSec{}           = []
 egetDocSec (StkhldrSec s)       = egetStk s
 egetDocSec (GSDSec g)           = egetGSD g
 egetDocSec (SSDSec s)           = egetSSD s
-egetDocSec (ReqrmntSec r)       = egetReq r
-egetDocSec (LCsSec l)           = egetLcs l
-egetDocSec LCsSec'{}            = [] -- likely changes can't lead to Expr?
-egetDocSec (UCsSec u)           = egetUcs u
+egetDocSec ReqrmntSec{}         = [] -- requirements can't lead to Expr?
+egetDocSec LCsSec{}             = [] -- likely changes can't lead to Expr?
```

Comment:
Are you absolutely sure of this? I am not.

Reply:
There is a requirement in GlassBR that has an `Expr` (which might be removed in the future) so that (at least for now) is wrong. I think I should assume that `Expr`s can show up in the changes as well, as there isn't any reason I can think of as to why they couldn't. Does this sound good @JacquesCarette?

Reply:
Yes. So the code should be adjusted to traverse things properly.

Reply:
I'm going to say to hold off on this (the LCs and UCs traversals). I have this fixed in a local branch which I'm in the process of cleaning up. No point in doubling up on the work. 😄

Reply:
@Mornix Should I still fix the requirements traversal?

Reply:
Already taken care of as well.

Reply:
Works for me!